x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -4,12 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
 import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import * as React from 'react';
 import { Provider as ReduxStoreProvider } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
 import { mountWithIntl } from 'test_utils/enzyme_helpers';
 
+import moment = require('moment');
+import { HostItem } from 'x-pack/plugins/secops/public/graphql/types';
```

Comment:
You want this to be a relative path such as `../../` rather than pulling in `x-pack` like this.